misc/librenms.service.scl

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=LibreNMS SNMP Poller Service
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/scl enable rh-python36 -- /opt/librenms/librenms-service.py -v
```

Comment:
Can you explain why you have included this?

Reply:
When a software collections package is installed, it's not enabled globally as they conflict with packages in the base system.

There's a magic incantation to switch it on (scl enable <package name>). To save the user having to modify the other unit, I've included this one and added 'os specific instructions' to the docs.

And I've just noticed that the 'os specific instructions' didn't get pushed. Now fixed.